backend/src/routes/sampleRoute.ts

```diff
@@ -0,0 +1,16 @@
+import * as express from 'express';
+import SampleController from '../controllers/sampleController';
+
+const sampleRoutes = (sampleController: SampleController, router) => {
```

Comment:
taka drobna zmiana...
`router: express.Router`

Reply:
hm, ja to mialem u siebie ale zapomniałem zapisać pliku :D
![image](https://user-images.githubusercontent.com/74790330/115613219-6d9e3b00-a2ec-11eb-966b-4b46e1854839.png)
juz dodane